CONTRIBUTING.md

```diff
@@ -41,90 +41,95 @@ Unit tests are nice for atomic components, but since jdbi is a complex ecosystem
+# Development Setup
 
-### Eclipse IDE
-
-* Window &rarr; Preferences &rarr; Java &rarr; Compiler
-* Section *Classfile Generation*
-* Check box *Add variable attributes to generated class files (used by the debugger)*
-* Check box *Store information about method parameters (usable via reflection)*
-* Click Apply and close dialog
+Most modern IDEs configure themselves correctly by importing the Jdbi repository as an Apache Maven project. If necessary, install support for Apache Maven first.
 
 ## Code Style / Formatting Rules
 
 The project uses a set of code style and formatting rules. These are enforced by Checkstyle and PMD as part of the build cycle.
-We ask that you configure the project in your development environment to support these rules as closely as possible.
 
-The JDBI Eclipse formatting preferences are available for download [here](internal/policy/src/main/resources/ide/jdbi-eclipse-formatter.xml).
+We do not review or merge PRs that do not pass our pre-merge checks.
 
 ### IntelliJ IDEA
 
-Import the file with the 'Adapter for Eclipse Code Formatter'. Additional info is available [here](https://plugins.jetbrains.com/plugin/6546-adapter-for-eclipse-code-formatter).
+* [IntelliJ IDEA formatting rules](https://github.com/jdbi/jdbi/blob/master/ide-support/intellij/jdbi.xml).
 
 ### Eclipse IDE
 
-Import these settings via Preferences &rarr; Java &rarr; Code Style &rarr; Formatter &rarr; Import... and activate them for all JDBI modules.
+* [Eclipse formatting rules](https://github.com/jdbi/jdbi/blob/master/ide-support/eclipse/jdbi-eclipse-formatter.xml).
+
+Import these settings via Preferences &rarr; Java &rarr; Code Style &rarr; Formatter &rarr; Import... and activate them for all Jdbi modules.
+
+* [Eclipse import order](https://github.com/jdbi/jdbi/blob/master/ide-support/eclipse/jdbi-eclipse.importorder).
 
-## Java Import Statements
+```
+#Organize Import Order
+0=java
+1=javax
+2=
+3=\#java
+4=\#javax
+5=\#
+```
+
+* Open your project's properties
+* Go to Preferences &rarr; Java &rarr; Code Style &rarr; Organize Imports
+* Click the *Import...* button and select the file you previously created
+* Set both text boxes *Number of [static] imports needed for .* * to a large value such as 1000, effectively turning wildcard/star imports off
+* Close the dialog
+* Reorganize imports of modified source files using these rules before any commit
+
+### Java Import Statements
 
 We enforce this order of imports:
 
 ```
 java.*
+
 javax.*
+
 *
+
 static java.*
+
 static javax.*
+
 static *
 ```
 
-A blank line is required between each group.
-Imports in a group must be ordered alphabetically.
+A blank line is required between each group. Imports in a group must be ordered alphabetically.
 
 Wildcard (aka star) imports e.g. `import org.apache.*;`, including static imports, are not allowed.
 
 Javadoc may not cause an import statement i.e. use FQCN in Javadoc unless the import statement is already caused by code.
```

Comment:
in hindsight, not everyone might understand _FQCN_. I would suggest to change into _fully qualified class name (FQCN)_

Reply:
+1